### PR TITLE
init for issue #4

### LINKED ISSUE #4: create initial files
Create the following files:

-  ./index.html (populate with [basic example](https://www.w3schools.com/html/html_basic.asp) for now)
- ./github/workflows/publish.yml (file with just 1 "workflow job" to build and publish Docker image to ghcr.io - see Ishango_Assessments repo as example)
- ./Dockerfile (see Ishango_Assessments repo as example)

let's try to deploy the Docker image on Digital Ocean Virtual Machine and run the basic html file there.
Once that works we can add more complexity.

### PR DESCRIPTION
**Pull request by oliverangelil**

### CODE REVIEW

File: .github/publish.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['release']
```
Comment (oliverangelil) [inline .github/publish.yml]: this should be the `development` branch.
we have no branch called `release` in this repo. 

You should also add:
```
pull_request:
    branches: [ development ]
```
    
  see example here:
  https://github.com/Ishangoai/Ishango_Assessments/blob/development/.github/workflows/publish.yml

File: dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM python:latest
```
Comment (oliverangelil) [inline dockerfile]: this shouldn't be be needed. this repo doesn't need python (at least not yet)
However we'll need a web server to run the .html file, so let's pull a copy of nginx from dockerhub. 
`FROM nginx:latest`

File: dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# Set the working directory for the app
+# WORKDIR /
+
+# Copy the app from the current folder to the working directory
+COPY . .
+
+# Install the required libraries and the module itself (necessary
+# to be able to run the tests with pytes)
+#RUN pip install -r ./requirements.txt
+#RUN pip install -e .
```
Comment (oliverangelil) [inline dockerfile]: you can remove these 2 lines, since there's no python.

File: dockerfile [hunk 4]
```diff
@@ -0,0 +1,14 @@
+FROM python:latest
+
+# Set the working directory for the app
+# WORKDIR /
+
+# Copy the app from the current folder to the working directory
+COPY . .
```
Comment (oliverangelil) [inline dockerfile]: Let's copy the index.html file into the location in the web server where files are hosted:
`COPY ./index.html /usr/share/nginx/html/index.html`

File: dockerfile [hunk 5]
```diff
@@ -0,0 +1,14 @@
+FROM python:latest
+
+# Set the working directory for the app
+# WORKDIR /
```
Comment (oliverangelil) [inline dockerfile]: you can remove this line

**Comment by oliverangelil** [comment]
@luelhagos @theoaid , once you have made changes based on my comments, you will see the docker image being pushed to the Github Container Registery. The workflow file should automate this whenever you push code. 

image

File: .github/publish.yml (hunk 1 above)
Comment (theoaid) [reply .github/publish.yml]: Yh, agree on your comment @Oliver, this attached [link](https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions) gives more information on this.

**Review by theoaid — APPROVED** [review]

**Pull request merged**